R/connecting.R

```diff
@@ -159,8 +206,13 @@ runJuliaServer <- function(port = 11980, multiclient = TRUE) {
 
    # start Julia server in background
    juliaexe <- getJuliaExecutablePath()
+
+   startupArgs <- Sys.getenv('JULIACONNECTOR_JULIAARGS')
+   if (startupArgs != '') startupArgs <- paste(startupArgs, '-- ')
```

Comment:
Are single dash arguments not allowed

Reply:
If I understood Julia docs well (https://docs.julialang.org/en/v1/manual/getting-started/), if you have startup options then you need to put a double dash before passing files, with the following pattern:
julia [switches] -- [programfile] [args...]